obspy/io/ascii/core.py

```diff
@@ -44,7 +44,7 @@
 from obspy.core.util import AttribDict, loadtxt
 
 
-HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %d sps, %.26s, %s, %s, %s\n"
+HEADER = "TIMESERIES %s_%s_%s_%s_%s, %d samples, %s sps, %.26s, %s, %s, %s\n"
 
```

Comment:
In case it was unclear earlier, `%lg` is the same as `%g` in Python:

> A length modifier (h, l, or L) may be present, but is ignored as it is not necessary for Python – so e.g. %ld is identical to %d.